app/controllers/admins_controller.rb

```diff
@@ -64,9 +60,8 @@ def verify
     if voter.save
       send_email = params[:send_email] == "true"
       if send_email
-        # Will need to be replaced with deliver_now
         begin
-          UserMailer.voter_verified(voter, event_year).deliver
+          UserMailer.voter_verified(voter, event_year).deliver_now
```

Comment:
Sorry, I'm being confusing because there's the production server and my local machine, and they are different.  The production server is still only pulling from the prod-2017 branch, which does not have this update.  I only merged the vote fix into prod-2017.